dali/kernels/imgproc/color_manipulation/equalize/lut.cu

```diff
@@ -85,13 +85,14 @@ void LutKernelGpu::Run(KernelContext &ctx, const TensorListView<StorageGPU, uint
   sample_descs_.clear();
   for (int sample_idx = 0; sample_idx < histogram.num_samples(); sample_idx++) {
     auto ranges = sequence_utils::unfolded_views_range<1>(lut[sample_idx], histogram[sample_idx]);
-    for (auto &&[lut, hist] : ranges) {
-      sample_descs_.push_back({lut.data, hist.data});
+    for (auto &&[lu, bin] : ranges) {
+      sample_descs_.push_back({lu.data, bin.data});
```

Comment:
Used element names
`lu` for lookup (from lookup table)
`bin` for a historgram element